src/core/lib/security/credentials/tls/grpc_tls_certificate_provider.h

```diff
@@ -132,7 +132,10 @@ class FileWatcherCertificateProvider final
   // information.
   std::map<std::string, WatcherInfo> watcher_info_;
 };
-
```

Comment:
nit: Add a blank line before and after the function.

Reply:
Noted, thanks.